src/main/resources/application.conf

```diff
@@ -81,6 +81,8 @@ metrics {
     port = ${?DATADOG_AGENT_PORT}
     tracked_consumer_group = []
     tracked_consumer_group =  ${?DATADOG_CONSUMER_GROUPS}
+    removeTagsFromMetricName = false
+    removeTagsFromMetricName = ${?REMOVE_TAGS_FROM_METRIC_NAME}
```

Comment:
Following the pattern again, would it make sense to prefix this env variable with `DATADOG_`, as it seems to be DataDog-specific setting?

Reply:
👍